caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4/istio-proxyv2:1.5.4 -->
+
+<image schemaversion="6.9" name="caasp-istio-proxyv2-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Istio proxyv2 running on SLE15 SP1 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/istio-proxyv2"
+        tag="1.5.4"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/cilium-envoy"/>
```

Comment:
The entrypoint in the upstream image is different, it is:
            "Entrypoint": [
                "/usr/local/bin/pilot-agent"
            ],

Reply:
@manuelbuil Thats the entrypoint for pilot image. This kiwi file is for proxyv2 image and cilium-envoy is installed in /usr/bin

Reply:
Not exactly. Remember the proxyv2 container executes two processes:

```
4     0     1     0  20   0 153468 47844 -      Ssl  ?          0:01 /usr/local/bin/pilot-agent proxy router --domain istio-system.svc.cluster.local --proxyLogLevel=warning --proxyComponentLogLevel=misc:error --
4     0    15     1  20   0 169572 49796 SyS_ep Sl   ?          0:00 /usr/local/bin/envoy -c /etc/istio/proxy/envoy-rev0.json --restart-epoch 0 --drain-time-s 45 --parent-shutdown-time-s 60 --service-cluster ist
```
I can confirm you that the upstream image `docker.io/cilium/istio_proxy` has the entrypoint `/usr/local/bin/pilot-agent`

Reply:
If I understand correctly, pilot-agent is the one triggering cilium-envoy. We need to investigate how exactly it does this

Reply:
I can confirm this. pilot-agent expects the envoy binary to be `/usr/local/bin/envoy` ==> https://github.com/istio/istio/blob/master/pkg/config/constants/constants.go#L46.

I suggest doing `ln -s /usr/bin/cilium-envoy /usr/local/bin/envoy`

Reply:
@manuelbuil Updated the entrypoint to use pilot-agent and the cilium-envoy is copied to /usr/local/bin/envoy in the package. Thanks.